gapic/src/main/com/google/gapid/views/CommandExperiments.java

```diff
@@ -0,0 +1,96 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.views;
+
+import com.google.gapid.models.Analytics.View;
+import com.google.gapid.models.Models;
+import com.google.gapid.models.ProfileExperiments;
+import com.google.gapid.proto.service.Service.ClientAction;
+import com.google.gapid.widgets.DialogBase;
+import com.google.gapid.widgets.Theme;
+import com.google.gapid.widgets.Widgets;
+
+import org.eclipse.jface.window.Window;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Shell;
+
+import java.util.logging.Logger;
+
+public class CommandExperiments {
+  private static final Logger LOG = Logger.getLogger(CommandEditor.class.getName());
+  private final Models models;
+  private final Theme theme;
+
+  public CommandExperiments(Models models, Theme theme) {
+    this.models = models;
+    this.theme = theme;
+  }
+
+  public void showExperimentsPopup(Shell parent) {
+    models.analytics.postInteraction(View.Commands, ClientAction.ShowExperiments);
+    ExperimentsDialog dialog = new ExperimentsDialog(parent, models, theme);
+    if (dialog.open() == Window.OK) {
+      models.analytics.postInteraction(View.Commands, ClientAction.UpdateExperiments);
+      models.profile.updateExperiments(dialog.newExperiments);
+    }
+  }
+
+  public ProfileExperiments getExperiments() {
+    return this.models.profile.getExperiments();
+  }
+
+  public void SetDisableAF(boolean selected) {
+    models.analytics.postInteraction(View.Commands, ClientAction.DisableAF);
+    models.profile.updateExperiments(new ProfileExperiments(selected));
+  }
+
+  private static class ExperimentsDialog extends DialogBase {
+    private final Models models;
+    private Button disableAFControl;
+    public ProfileExperiments newExperiments;
+
+    public ExperimentsDialog(Shell parentShell, Models models, Theme theme) {
+      super(parentShell, theme);
+      this.models = models;
+    }
+
+    @Override
+    public String getTitle() {
+      return "Profile Experiments";
+    }
+
+    @Override
+    protected Control createDialogArea(Composite parent) {
+      Composite area = (Composite)super.createDialogArea(parent);
+      Composite container = Widgets.createComposite(area, new GridLayout(2, false));
+      container.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
+      Widgets.createLabel(container, "Disable Anisotropic Filtering");
+      disableAFControl = Widgets.createCheckbox(container, models.profile.getExperiments().getShouldDisableAnisotropicFiltering());
```

Comment:
There is a `createCheckbox` that takes in a label for the checkbox. It is preferred over this approach, as it makes the label clickable to toggle the checkbox. Just set the layout data on the control to span two columns in the layout.